azurecompute/src/main/java/org/jclouds/azurecompute/compute/options/AzureComputeTemplateOptions.java

```diff
@@ -54,6 +54,7 @@
    protected String networkSecurityGroupName;
    protected String reservedIPName;
    protected Boolean provisionGuestAgent;
+   protected Boolean winrmUseHttps;
```

Comment:
Thank you very much for the suggestion.
Looks good however I am more for the simplest approach.